lib/ipinfo/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module IPinfo
-    VERSION = '1.1.0'
+    VERSION = '1.1.1'
```

Comment:
We'll need a major version bump now as we have a backwards incompatible change - the meaning of the settings we updated is now to expect an in-memory map and not a file.

Reply:
Done.